qa/common/gen_qa_models.py

```diff
@@ -493,12 +495,29 @@ def create_plan_dynamic_rf_modelfile(models_dir, max_batch, model_version,
 
     in0 = network.add_input("INPUT0", trt_input_dtype, input_with_batchsize)
     in1 = network.add_input("INPUT1", trt_input_dtype, input_with_batchsize)
+
+    # uint8 must be converted before any operation
```

Comment:
Can you elaborate more about why we are casting uint8 into float32?
Such as TRT identity layer does not support uint8 datatype yet.

Reply:
Yes. Added "TRT uint8 cannot be used to represent quantized floating-point value yet".